pkg/controller/volume/scheduling/scheduler_binder_test.go

```diff
@@ -1046,12 +1023,6 @@ func TestAssumePodVolumes(t *testing.T) {
 			expectedBindings:      []*bindingInfo{makeBinding(unboundPVC, pvNode1aBound)},
 			expectedProvisionings: []*v1.PersistentVolumeClaim{},
 		},
-		"claimref-failed": {
```

Comment:
don't we still have cases that cannot extract a claimref? should we exercise one of those here?

Reply:
Well - technically it can fail only if the type is not registered in scheme. And it's not really possible to inject scheme without PVC types registered here...